diff --git a/modules/cctbx_project/libtbx/SConscript b/modules/cctbx_project/libtbx/SConscript
index b64decdf1a..2d4bfb9a71 100644
--- a/modules/cctbx_project/libtbx/SConscript
+++ b/modules/cctbx_project/libtbx/SConscript
@@ -401,8 +401,7 @@ For newer versions, please adjust this SConscript.
       "/bigobj", # for heavy use of template libraries
       "/wd4996", # warning C4996: ... was declared deprecated
       "/wd4068", # warning C4068: unknown pragma
-      "/Z7", # produce program database file containing debugging symbols.
-             # Harmless for release builds
+      "/DBOOST_TIMER_ENABLE_DEPRECATED", # turn off Boost.timer deprecation
       "/Zm800"]
     # include header files from cctbx_project/msvc9.0_include for VS 2008
     if (env_base['MSVC_VERSION'] == '9.0'):
